src/Widgets/PartitionMenu.vala

```diff
@@ -250,39 +250,36 @@ public class Installer.PartitionMenu : Gtk.Popover {
     }
 
     private void check_values (SetMount set_mount) {
-        if (values_ready ()) {
-            update_values (set_mount);
+        if (!use_partition.active) {
+            partition_bar.icon = null;
+            return;
+        }
+
+        if (use_as.active == 4 && !custom.text.has_prefix ("/")) {
+            partition_bar.icon = new ThemedIcon ("dialog-warning-symbolic");
+            partition_bar.tooltip_text = _("Custom value must begin with /");
+            return;
         }
-    }
 
-    private void update_values (SetMount set_mount) {
         var mount = get_mount ();
-        var filesystem = mount == "swap"
-            ? InstallerDaemon.FileSystem.SWAP
-            : get_file_system ();
+        var filesystem = mount == "swap" ? InstallerDaemon.FileSystem.SWAP : get_file_system ();
 
-        string? error = null;
         try {
             set_mount (new Installer.Mount (
                 partition_path,
                 parent_disk,
                 mount,
                 partition_bar.get_partition_size (),
                 (format_partition.active ? InstallerDaemon.MountFlags.FORMAT : 0)
-                    + (is_lvm ? InstallerDaemon.MountFlags.LVM : 0),
+                + (is_lvm ? InstallerDaemon.MountFlags.LVM : 0),
                 filesystem,
                 this
             ));
-        } catch (GLib.Error why) {
-            error = why.message;
-        }
-
-        partition_bar.icon = new ThemedIcon (
-            error == null ? "process-completed-symbolic" : "dialog-warning-symbolic"
-        );
 
-        if (error != null) {
-            partition_bar.tooltip_text = error;
+            partition_bar.icon = new ThemedIcon ("process-completed-symbolic");
+        } catch (GLib.Error e) {
+            partition_bar.icon = new ThemedIcon ("dialog-warning-symbolic");
+            partition_bar.tooltip_text = e.message;
```

Comment:
It looks like the tooltip text remains after resolving the error, e.g. change not to use the partition or change to valid usage. This also happens on main so maybe we may want to fix it in another PR?

![VirtualBox_elementary OS 8_01_10_2024_23_07_16](https://github.com/user-attachments/assets/21734a05-31a0-4975-a70c-bbfa9cab9886)

![VirtualBox_elementary OS 8_01_10_2024_23_07_37](https://github.com/user-attachments/assets/f318e90c-9887-4884-a700-b2411430652a)

Reply:
Fixed!